luminance-webgl/src/webgl2/state.rs

```diff
@@ -614,6 +618,12 @@ impl fmt::Display for StateQueryError {
         f.write_str("cannot retrieve blending destination factor (alpha)")
       }
 
+      StateQueryError::CannotRetrieveRequiredWebGLExtensions(ref extensions) => {
+        f.write_str("missing WebGL extensions: [")?;
+        f.write_str(extensions.join(", ").as_str())?;
+        f.write_str("]")
```

Comment:
You can simplify this with:

```suggestion
        write!(f, "missing WebGL2 extensions: [{}]", extensions.join(", ").as_str())
```

Reply:
I like it!  Done.